src/components/FAQAccordion/index.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { useState } from 'react';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import styles from './style.module.scss';
+import Typography from '../Typography';
+
+interface FaqProps {
+  data: { question: string; answer: string | any }[];
+}
+
+export default function FAQ({ data }: FaqProps) {
```

Comment:
The file, component, and type names all differ. Maybe you should choose one for consistency

Reply:
Oh ya it's spaghetti because I ripped it off of the old site